scipy/optimize/_particleswarmoptimization.py

```diff
@@ -38,151 +37,186 @@ def particle_swarm_optimization(func, bounds, initial_bounds=None, args=None,
         required to have ``len(bounds) == len(x)``. ``len(bounds)`` is used
         to determine the number of parameters in ``x``.
     initial_bounds : sequence or `Bounds`, optional
-        Initial bounds for variables, inside the global search bounds. If it is 
-        not specified, the initial bounds are settle as the global bounds. 
+        Initial bounds for variables, inside the global search bounds. If it is
+        not specified, the initial bounds are settle as the global bounds.
         There are two ways to specify the initial bounds:
         1. Instance of `Bounds` class.
         2. ``(min, max)`` pairs for each element in ``x``, defining the finite
```

Comment:
I would only accept the `Bounds` class for this, it's a more modern way of doing things.